eslint.config.js

```diff
@@ -3,21 +3,17 @@ import {fileURLToPath} from "node:url"
 
 import {includeIgnoreFile} from "@eslint/compat"
 import js from "@eslint/js"
-import prettier from "eslint-config-prettier"
```

Comment:
`eslint-config-prettier` serves purely to turn off formatting/layout rules that conflict with Prettier. None of the plugins in this file turn on any of those rules, so I don't think it's needed anymore.

https://www.joshuakgoldberg.com/blog/you-probably-dont-need-eslint-config-prettier-or-eslint-plugin-prettier/#eslint-config-prettier

Reply:
Worked my way through the configuration one by one and you're right. I think I can delete it now!

I wrote the original configuration before I integrated `prettier`, so I had some leftover style rules in `eslint`.